Chummer/Backend/Equipment/Drugs.cs

```diff
@@ -790,8 +790,8 @@ public int Duration
             {
                 if (_intCachedDuration != int.MinValue)
                     return _intCachedDuration;
-                if (string.IsNullOrWhiteSpace(_strDuration))
-                    return _intCachedDuration = 0;
+                //if (string.IsNullOrWhiteSpace(_strDuration))
+                //    return _intCachedDuration = 0;
```

Comment:
Rather than disabling this shortcut version of caching drug duration when there are no extra duration-modifying blocks applied, I think it would be better to just make sure it properly detects when there is indeed nothing present that would require expensive duration calculations to be done. Assuming you can do that fast enough, of course; if you can't then yeah, just remove this code.